src/test/java/easytests/api/v1/controllers/SubjectsControllerTest.java

```diff
@@ -133,9 +133,85 @@ public void testListForbidden() throws Exception {
     /**
      * create
      */
-    /**
-     * update
-     */
+    @Test
+    public void testUpdateSuccess() throws Exception {
+        final SubjectModelInterface subjectModel = subjectsSupport.getModelFixtureMock(0);
+
+        when(this.subjectsService.find(any(), any())).thenReturn(subjectModel);
+        when(this.acl.hasAccess(any(UserModelInterface.class))).thenReturn(true);
+
+        mvc.perform(put("/v1/subjects")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(new JsonSupport()
+                        .with(id, 1)
+                        .with(name, "newSubject")
+                        .with(description, "newDescription")
+                        .with(user, 1)
+                        .build()
+                ))
+                .andExpect(status().isOk())
+                .andExpect(content().string(""))
+                .andReturn();
+
+        verify(this.subjectsService, times(1)).save(subjectModel);
```

Comment:
nope, we should verify, that subject model have been passed into save method have fields that we pass from request. i.e. newSubject newDescription.

Reply:
done